Fabric.Identity.API/Startup.cs

```diff
@@ -192,13 +213,12 @@ public void ConfigureServices(IServiceCollection services)
         public void Configure(
             IApplicationBuilder app,
             IHostingEnvironment env,
-            ILoggerFactory loggerFactory,
             IDbBootstrapper dbBootstrapper)
         {
             if (env.IsDevelopment())
             {
                 app.UseDeveloperExceptionPage();
-                _loggingLevelSwitch.MinimumLevel = LogEventLevel.Verbose;
+                LogFactory.LoggingLevelSwitch.MinimumLevel = LogEventLevel.Verbose;
```

Comment:
should this be something exposed in web.config or appsettings.json to allow the type of logging?